tests/unit/resources/__snapshots__/methods.test.ts.snap

```diff
@@ -45,6 +47,7 @@ List [
       "size1x": "https://www.mollie.com/images/payscreen/methods/banktransfer.png",
       "size2x": "https://www.mollie.com/images/payscreen/methods/banktransfer%402x.png",
     },
+    "mode": undefined,
```

Comment:
This is because the `Method` class extends the `Model` class, which has a `mode` property. I added a hack to kind-of fix this, but the inheritance chain is not accurate here.